core/src/test/scala/integration/kafka/server/MultipleListenersWithSameSecurityProtocolWithMultipleJaasContextTest.scala

```diff
@@ -0,0 +1,45 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements.  See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership.  The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License.  You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+package integration.kafka.server
+
+import java.util.Properties
+
+import kafka.api.CustomKafkaServerSasl
+import kafka.server.MultipleListenersWithSameSecurityProtocolBaseTest
+import org.apache.kafka.common.network.ListenerName
+
+
+class MultipleListenersWithSameSecurityProtocolWithMultipleJaasContextTest extends MultipleListenersWithSameSecurityProtocolBaseTest{
```

Comment:
@baluchicken Yes, that is better. The filename could be the same as the shorter classname too?